spec.html

```diff
@@ -39227,69 +38992,84 @@ <h1>AsyncGeneratorRequest Records</h1>
       </emu-clause>
 
       <emu-clause id="sec-asyncgeneratorstart" aoid="AsyncGeneratorStart">
-        <h1>AsyncGeneratorStart ( _generator_, _generatorBody_ )</h1>
-        <p>The abstract operation AsyncGeneratorStart takes arguments _generator_ and _generatorBody_. It performs the following steps when called:</p>
+        <h1>AsyncGeneratorStart ( _generator_, _generatorBody_, _generatorBrand_ )</h1>
+        <p>The abstract operation AsyncGeneratorStart takes arguments _generator_, _generatorBody_ (a Parse Node or an Abstract Closure with no parameters), and _generatorBrand_. It performs the following steps when called:</p>
         <emu-alg>
-          1. Assert: _generator_ is an AsyncGenerator instance.
+          1. Perform ? AsyncGeneratorValidate(_generator_, _generatorBrand_).
```

Comment:
I don't think this is necessary, is it? There's only two callsites of this method, both of which are immediately preceded by steps setting the `[[GeneratorBrand]]` slot to the value checked for here.